cmd/traffic_manager/MgmtHandlers.cc

```diff
@@ -44,60 +44,6 @@
 
 int aconf_port_arg = -1;
```

Comment:
Do we still need this aconf_port_arg? I *think* this is a leftover naming from when the synthetic port also served the proxy autoconf files.

Reply:
Ok.  Wasn't clear what that was.